ansible/stage2.yml

```diff
@@ -1,13 +1,32 @@
 ---
-- name: Execute getbestchainlock
+
+#We need to check first to ensure mixing is not enabled from a previous run, if it tries to do it again it will fail (thus not idempotent)
+- name: Check mixing status and start mixing if not enabled
   hosts: wallet_nodes
+  become: true
   tasks:
```

Comment:
Can we move these tasks into the `mixing` role? In `deploy.yml` we only include roles in the top level, which makes handling conditions in stage 2 easier (e.g. when we add proposals next).

Also the step name makes it seem like these tasks will only run in case a condition is satisfied, but there is no condition specified here.